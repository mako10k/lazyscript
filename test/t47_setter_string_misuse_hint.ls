!{
  ~ns <- (~~nsnew0);
  # misuse: string key "__set" should hint correct usage
  ((~ns "__set") .x 1)
};
