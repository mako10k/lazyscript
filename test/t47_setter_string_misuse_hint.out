E: #err "namespace: use (~ns __set) or (~ns .__set) for setter"
